docs/source/usage.rst

```diff
@@ -62,15 +62,17 @@ Output Arguments
       -hap                  Flag to call and write out the haplotypes.
       -geno                 Flag to call and write out the genotypes.
       -geno_threshold [GENO_THRESHOLD [GENO_THRESHOLD ...]]
-                            Genotype calling threshold(s). Multiple space
-                            separated values allowed. Use .3 for best guess
-                            genotype.
+                            Genotype calling threshold(s). Multiple space separated values allowed.
+                            Any value less than 1 / 3 would be replaced by 1 / 3.
```

Comment:
@AprilYUZhang: Change `Any value less than 1 / 3 would be replaced by 1 / 3.` to `Value less than 1 / 3 will be replaced by 1 / 3.`

Reply:
have updated.